### PR TITLE
Krish issues

### PR DESCRIPTION
**Pull request by krish7777**
Implemented basic auth, solved the navbar bugs

### CODE REVIEW

**Comment by arpit1912** [comment]
@krish7777  can you resolve the conflicts with the main branch, so that it will be easy to check

File: frontend/src/components/Auth/Login.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+
+    onSubmit = values => {
+        this.props.loginUser(values);
+    };
+
+    render() {
+        const { errors } = this.state;
+
+        // return (
```
Comment (arpit1912) [inline frontend/src/components/Auth/Login.js]: remove this comments if not needed

File: frontend/src/components/Auth/Register.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        if (values.password != values.password2)
+            this.openNotificationWithIcon('warning', 'Passwords are not matching')
+        else
+            this.props.registerUser(values, this.props.history);
+    };
+
+    render() {
+        const { errors } = this.state;
+
+        // return (
```
Comment (arpit1912) [inline frontend/src/components/Auth/Register.js]: ditto

**Review by arpit1912 — CHANGES_REQUESTED** [review]
Also do add the way to run it, maybe a brief explaining what is happening can also help

**Comment by krish7777** [comment]
Made all the requested changes. 
To run using a single command,

npm run dev

Make sure nodemon is installed globally, or write npm i -g nodemon

**Comment by krish7777** [comment]
Also for sample login you can use.

email:        demo@gmail.com
password:  password

Or you can create users.

Once logged in, token will be stored, so no need to login again , unless you logout.

**Review by arpit1912 — APPROVED** [review]

**Pull request merged**